README.md

```diff
@@ -1,38 +1,19 @@
 # JavaScript Basics in Express
 
-## Getting started
-
-Ensure your Visual Studio Code application is up to date, and that you have the [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) installed and enabled.
-
-Run the commands below inside your `Projects` folder.
-
-```bash
-git clone git@github.com:MCRcodes/javascript-basics-express.git
-cd your-project-folder-name
-npm install
-```
-
-## Copy over your JavaScript Basics code
-
-:rotating_light: **Important!**
-
-This exercise makes use of your previous `javascript-basics` code. Copy the files from that project's `src/` folder into a new `lib/` directory in this repository's `src` directory. You should end up with the following file structure in this project:
-
-```
-src
-├── app.js
-└── lib
-    ├── arrays.js
-    ├── booleans.js
-    ├── numbers.js
-    ├── objects.js
-    └── strings.js
-```
-
-## Running tests
-
-You can run tests with the `npm test` command.
-
-## Remember: READMEs are important!
-
-When you're finished, be sure to rewrite this one so it's more specific to your project.
+## Learning objectives
+
+* Explain what a Web API is and why they are necessary.
+* Use and create a web server using Express.
+* Create routes in an Express application to direct requests to a controller function.
+* Understand what is meant by the term middleware.
+* Retrieve route params, query params and body params from a HTTP request using Express.
+* Send a HTTP response with a custom status code and body using Express.
+* Explain the difference between an end-to-end (E2E) test and a unit test.
+
+## Concepts
+
+* Web Servers and Web API's.
+* HTTP Requests and Responses.
+* Routing.
+* Controller functions.
+* Middleware.
```

Comment:
It would be great if you could include a table of the routes this uses here, for example

```
## Endpoints
### Strings

| HTTP method | Method      | Endpoint | Description |
| ----------- | ----------- |----------- | ----- |
| GET         | sayHello(string) | /strings/hello/:string | Returns the string "Hello, " concatenated with the given string. |
| GET         | uppercase(string) | /strings/upper/:string | Returns the given string converted to all uppercase. |
```
You could create a new table for each grouping like this, adding whatever info you like. Have a look at [this guide](https://www.markdownguide.org/extended-syntax/) for more info on the formatting options possible